runtime/common/src/lib.rs

```diff
@@ -81,6 +85,7 @@ where
 	fn get(currency_id: &CurrencyId) -> Balance {
 		match currency_id {
 			CurrencyId::Native => T::ExistentialDeposit::get(),
+			CurrencyId::Staking(StakingCurrency::BlockRewards) => T::ExistentialDeposit::get(),
```

Comment:
Here we were assuming `ED` for `StakingCurrency` was registered in the orml asset registry or, if not, it was `ED`. But the value was always `T::ExistentialDeposit::get()` which maps to `1 * MICRO_CFG`